### ISSUE TITLE
Critical changes to signup flow

### ISSUE THREAD

**Issue by ItsiW** [Couchers-org/couchers#980]
Some changes before we can start pushing it on marketing channels

- [x] Change to the date selector component so it's easier to change years
- [ ] Add ("press Enter to search") to map search descriptive text
- [ ] Make magnifying glass replicate hitting enter on map search input
- [ ] Remove adjustable radius, and set it at 1km (although leave this as adjustable on the edit profile page)
- [ ] New text field underneath map for "Display address" that is auto-populated with the location selection

**Comment by justjooshing** [comment]
The color contrast is also quite hard to read on mobile (edit: created new issue here [#983](https://github.com/Couchers-org/couchers/issues/983)
![image](https://user-images.githubusercontent.com/60824744/113581260-d31cd700-961e-11eb-8352-ee326e461ca9.png)

**Comment by justjooshing** [comment]
@ItsiW What's difficult re the ease of changing years atm?
Just the default date being 1900? I found it easy to type in a date on mobile, but first tried using the calendar and scrolling by month from 1900 is ridiculous haha

**Comment by ItsiW** [comment]
Looks like we'll have to change that link colour, could you put up an issue for that?

Yeah it's just that using the calendar widget is too difficult and I think most people would use that rather than typing. Even if it was 5 years away from your birthdate that's a lot of scrolling

**Comment by lucaslcode** [comment]
@justjooshing is working on year input. @aapeliv is working on map related issues.

**Comment by ItsiW** [comment]
Looking at the new date selector component, could we make it so that it opens up to the year selection? This removes an extra click and I feel the year selector option is slightly less than obvious

**Comment by justjooshing** [comment]
Alright, just pushed that change, for the birthday signup flow only (the others will still open on date). Let me know what you think once it's merged, and we can adjust if we want @ItsiW 
https://github.com/Couchers-org/couchers/pull/991
